src/readers/src/T2D.h

```diff
@@ -3,6 +3,8 @@
 #include <numeric>
 #include <vector>
 
+#include <map>
+
 namespace {
```

Comment:
Do not use unnamed namespaces in header files - there is no any point to do that